tests/Games/NexusMods.Games.StardewValley.Tests/DependencyDiagnosticEmitterTests.cs

```diff
@@ -102,6 +117,19 @@ public async Task Test_RequiredDependencyIsOutdated()
         var dependent = requiredDependencyIsOutdatedMessageData.Dependent.ResolveData(ServiceProvider, Connection);
         dependent.AsLoadoutItem().ParentId.Should().Be(farmTypeManager.LoadoutItemGroupId, because: "Farm Type Manager is the dependent");
 
+        // Test disabled cases
+        await DisableItem(farmTypeManager);
+        
+        (await GetAllDiagnostics(loadout)).OfType<Diagnostic<Diagnostics.RequiredDependencyIsOutdatedMessageData>>()
+            .Should().BeEmpty(because: "The dependant Farm Type Manager is disabled");
+        
+        await EnableItem(farmTypeManager);
+        await DisableItem(contentPatcher);
+        
+        (await GetAllDiagnostics(loadout)).OfType<Diagnostic<Diagnostics.RequiredDependencyIsOutdatedMessageData>>()
+            .Should().BeEmpty(because: "The outdated dependency Content Patcher is disabled");
```

Comment:
This should just be `ShouldHaveNoDiagnostics`.

Reply:
Same as above, we get a different diagnostic if it is disabled.